blob/drivertest/drivertest.go

```diff
@@ -253,7 +253,7 @@ func testAttributes(t *testing.T, makeBkt BucketMaker) {
 		b, done := init(t)
 		defer done()
 
-		r, err := b.NewRangeReader(ctx, key, 0, 0)
+		r, err := b.NewRangeReader(ctx, key, 0, 1)
```

Comment:
Since the implementations for several providers are different for 0-length vs non, how about testing both? With your suggested change, we're no longer testing the 0-length read.

You could do this either by putting a loop around all of the t.Runs here, something like:

for _, readSize := []int64{0, 1} {
  t.Run(fmt.Sprintf("ReadLength%d", readSize), ... {
     t.Run("Size", ...
     t.Run("Attributes", ...
      ..
  }
}

Or, by adding a "readLength int" argument to testAttributes and calling it twice up above.

Reply:
Note that we do a second round of read a few seconds after to test the ModTime change, that one tests zero-length read. I think for this specific case the coverage is enough. WDYT?

Reply:
That's true for ModTime, but not the others (Size + ContentLength), so I think it's better to test both explicitly. I'd update the 2nd read for ModTime to be the same as the first one.